src/applications/vaos/services/appointment/transformers.js

```diff
@@ -0,0 +1,273 @@
+    }
+    case APPOINTMENT_TYPES.vaAppointment: {
+      const currentStatus = isVideoVisit(appointment)
+        ? appointment.vvsAppointments?.[0]?.status?.code
+        : appointment.vdsAppointments?.[0]?.currentStatus;
+
+      if (
+        (isPastAppointment &&
+          PAST_APPOINTMENTS_HIDE_STATUS_SET.has(currentStatus)) ||
+        (!isPastAppointment &&
+          FUTURE_APPOINTMENTS_HIDE_STATUS_SET.has(currentStatus))
+      ) {
+        return null;
+      }
+
+      const cancelled = CANCELLED_APPOINTMENT_SET.has(currentStatus);
+
+      return cancelled
+        ? APPOINTMENT_STATUS.cancelled
+        : APPOINTMENT_STATUS.booked;
+    }
+    default:
+      return APPOINTMENT_STATUS.booked;
+  }
+}
+/**
+ * Finds the datetime of the appointment depending on the appointment type
+ * and returns it as a moment object
+ *
+ * @param {Object} appt VAR appointment object
+ * @returns {Object} Returns appointment datetime as moment object
+ */
+function getMomentConfirmedDate(appt) {
+  if (isCommunityCare(appt)) {
+    const zoneSplit = appt.timeZone.split(' ');
+    const offset = zoneSplit.length > 1 ? zoneSplit[0] : '+0:00';
+    return moment
+      .utc(appt.appointmentTime, 'MM/DD/YYYY HH:mm:ss')
+      .utcOffset(offset);
+  }
+
+  const timezone = getTimezoneBySystemId(appt.facilityId)?.timezone;
+  const date = isVideoVisit(appt)
+    ? appt.vvsAppointments[0].dateTime
+    : appt.startDate;
+  return timezone ? moment(date).tz(timezone) : moment(date);
+}
+
+/**
+ * Returns url for user to join video conference
+ *
+ * @param {Object} appt VAR appointment object
+ * @returns {String} URL of video visit
+ */
+function getVideoVisitLink(appt) {
+  return appt.vvsAppointments?.[0]?.patients?.[0]?.virtualMeetingRoom?.url;
+}
+
+/**
+ * Returns appointment duration in minutes. The default is 60 minutes.
+ *
+ * @param {Object} appt VAR appointment object
+ * @returns
+ */
+function getAppointmentDuration(appt) {
+  const appointmentLength = parseInt(
+    appt.vdsAppointments?.[0]?.appointmentLength ||
+      appt.vvsAppointments?.[0]?.duration,
+    10,
+  );
+  return isNaN(appointmentLength) ? 60 : appointmentLength;
+}
+
+/**
+ * Builds participant and contained arrays for FHIR Appointment object which usually
+ * contain Location (Facility) and HealthcareService (Clinic) or video conference info
+ *
+ * @param {Object} appt  VAR appointment object
+ * @returns {Array} Array of participants of FHIR appointment
+ */
+function addParticipantAndContained(appt, transformed) {
+  const isVideo = isVideoVisit(appt);
+  let contained;
+
+  const participant = [
+    {
+      actor: {
+        reference: `Location/var${appt.facilityId}`,
+      },
+    },
+  ];
+
+  if (!isVideo) {
+    participant.push({
+      actor: {
+        reference: `HealthcareService/var${appt.clinicId}`,
+        display:
+          appt.clinicFriendlyName || appt.vdsAppointments?.[0]?.clinic?.name,
+        telecom: isVideoVisit(appt)
```

Comment:
Do we need this `isVideoVisit` check? It seems like we're only in this branch if it's not a video appointment